CRM/Contribute/Page/ContributionPage.php

```diff
@@ -182,6 +182,11 @@ public function &configureActionLinks() {
           'uniqueName' => 'pcp',
         ),
       );
+      $context = array(
+        'urlString' => $urlString,
+        'urlParams' => $urlParams,
+      );
+      CRM_Utils_Hook::tabset('civicrm/admin/contribute', self::$_configureActionLinks, $context);
```

Comment:
This is to add the child action link under `Configure` option for the tab created by the hook on  `Manage Contribution Pages` screen (civicrm/admin/contribute?reset=1). The call written in TabHeader.php only adds the tab which is seen while creating the contribution page.

The `urlString` is the url to which the newTab is to be appended as done for other links above, 
`urlParams` is the queryString for the same - eg `reset=1&action=update&id=1`.

I've [updated the wiki](http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_tabset) for the usage with which this link can be added under Configure option. As I see, [this is also done for Events](https://github.com/civicrm/civicrm-core/blob/master/CRM/Event/Page/ManageEvent.php#L178).